packages/syft/src/syft/service/worker/worker_pool.py

```diff
@@ -49,7 +50,7 @@ class WorkerHealth(Enum):
 @serializable()
 class SyftWorker(SyftObject):
```

Comment:
Can we keep version 2 SYftWorker and create a new class for version 3 and define migrations for them as well.

Reply:
Resolved by: https://github.com/OpenMined/PySyft/pull/8906/commits/2b27c5d69666cefeb5701130fce94d46168919c0